test/build.info

```diff
@@ -411,6 +412,20 @@ IF[{- !$disabled{tests} -}]
   INCLUDE[dtlstest]=../include ../apps/include
   DEPEND[dtlstest]=../libcrypto ../libssl libtestutil.a
 
+  IF[{- !$disabled{sock} -}]
+    PROGRAMS{noinst}=bio_write_test bio_read_test
+  ENDIF
+  IF[{- !$disabled{dgram} -}]
+    PROGRAMS{noinst}=bio_write_test bio_read_test
+  ENDIF
+  SOURCE[bio_write_test]=bio_write_test.c
+  INCLUDE[bio_write_test]=../include ../apps/include
+  DEPEND[bio_write_test]=../libcrypto libtestutil.a
+
+  SOURCE[bio_read_test]=bio_read_test.c
+  INCLUDE[bio_read_test]=../include ../apps/include
+  DEPEND[bio_read_test]=../libcrypto libtestutil.a
```

Comment:
Probably the SOURCE/INCLUDE/DEPEND lines can also be within the `IF[{- !$disabled{dgram} -}]` block.